source/css/_common/outline/sidebar/sidebar-nav.styl

```diff
@@ -1,12 +1,14 @@
 // Sidebar Navigation
 .sidebar-nav {
-  display: none;
+  --scroll-height: auto;
+  height: 0;
   margin: 0;
-  padding-bottom: 20px;
+  overflow: hidden;
   padding-left: 0;
+  transition: height $transition-ease;
 
   .sidebar-nav-active & {
-    display: block;
+    height: var(--scroll-height);
```

Comment:
Could we assume the line height as `2em` (`line-height: 2` from `body`), so that we can use `height: calc(2em + 1px)` (1px border-bottom) for pure CSS transitions?

Reply:
e8c1cc962b8893c230a99ac7487bff8685222a71